packages/palette/src/palette.story.tsx

```diff
@@ -26,13 +32,13 @@ const ColorBlock = styled('div')`
     font-size: 12px;
     text-align: center;
     padding: 3px 0.3rem;
-    color: ${props => readableColor(lighten(0.2, props['data-color']))};
-    background-color: ${props => lighten(0.2, props['data-color'])};
+    color: ${props => readableColor(lighten(0.2, props.color))};
+    background-color: ${props => lighten(0.2, props.color)};
     border-radius: 4px;
   }
 
   &:after {
-    content: attr(data-name);
+    content: "${props => props.name}";
```

Comment:
Why is this JS solution preferable to the baked-in CSS function?

Reply:
🤷‍♂ this was we can give it a type